fullstop-violation/fullstop-violation-jpa/src/main/java/org/zalando/stups/fullstop/rule/entity/RuleEntity.java

```diff
@@ -1,26 +1,19 @@
 package org.zalando.stups.fullstop.rule.entity;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.EntityListeners;
-import javax.persistence.Id;
-import javax.persistence.ManyToOne;
-import javax.persistence.Table;
-import javax.persistence.Version;
-import javax.validation.constraints.NotNull;
-
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
 import org.hibernate.validator.constraints.NotEmpty;
 import org.joda.time.DateTime;
+import org.joda.time.LocalDate;
 import org.springframework.data.annotation.CreatedBy;
 import org.springframework.data.annotation.CreatedDate;
 import org.springframework.data.annotation.LastModifiedBy;
 import org.springframework.data.annotation.LastModifiedDate;
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
-import org.zalando.stups.fullstop.domain.AbstractModifiableEntity;
 import org.zalando.stups.fullstop.domain.validation.groups.PersistenceOnly;
-import org.zalando.stups.fullstop.violation.entity.ViolationTypeEntity;
 
-import com.google.common.base.MoreObjects;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
 
 @Table(name = "rule_entity", schema = "fullstop_data")
```

Comment:
to be backward-compatible, we could create a completely new table (simply called "rule") and point this Entity to the new table. Otherwise we may run into trouble during deployment...